qiskit-superstaq/qiskit_superstaq/compiler_output.py

```diff
@@ -130,13 +161,15 @@ def read_json_aqt(  # pylint: disable=missing-param-doc
     """Reads out returned JSON from SuperstaQ API's AQT compilation endpoint.
```

Comment:
minor nit: Can you update SuperstaQ to Superstaq wherever applicable in the docstrings? For example:
```suggestion
    """Reads out returned JSON from Superstaq API's AQT compilation endpoint.
```